app/models/search/work_indexer.rb

```diff
@@ -26,8 +26,8 @@ def self.mapping
           tag: {
             type: "text"
           },
-          series_titles: {
-            type: "text"
+          series: {
+            type: "object"
```

Comment:
For the record, [arrays of inner object fields are flattened](https://www.elastic.co/guide/en/elasticsearch/reference/6.0/nested.html), meaning the association between id, title, and position is lost when a work is indexed. However, we're not querying for combinations e.g. series title AND series position, we're only querying for series title, and the other 2 fields are there for eventual use in work blurbs ([AO3-5760](https://otwarchive.atlassian.net/browse/AO3-5760)). So we don't need to use the nested data type.